spec/latest/core/index.html

```diff
@@ -823,6 +823,55 @@ <h3>Collections</h3>
       <i>first</i> and, perhaps more interestingly, <i>last</i>?</p>
   </section>
 
+  <section>
+    <h4>"Manages block"</h4>
+
+    <p>A manages block is a way to declare additional, implicit statements about
+      members of a <a href="#collections">collection</a>. Statements which may otherwise
+      be missing from the respective member resources inlined in a collection's
+      representation.</p>
+
+    <pre class="example nohighlight"
+         data-transform="updateExample"
+         title="Manages block describes relation with another resource">
+      <!--
+      {
+        "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+        "@id": "http://api.example.com/an-issue/comments",
+        "@type": "Collection",
+        ****"manages": {
+          "subject": "http://api.example.com/an-issue",
+          "property": "http://api.example.com/vocab#comment"
+        }****,
+        "member": [
+          {
+            "@id": "/comments/429"
+          }
+        ]
+      }
+      -->
+    </pre>
+
+    <p>In the above example, adding a <code>manages</code> node to the collection instructs the
+      client that every member of this collection is linked to the <code>subject</code>
+      by the <code>property</code>. It could be written a SPARQL triple pattern below, where
```

Comment:
A minor typo *"It could be written as a SPARQL triple pattern"*
btw thanks for this, earlier I went through all PRs related to *manages* block but couldn't understand much, now it makes perfect sense.